src/main/java/com/hashedin/redmask/service/SnowflakeMaskingService.java

```diff
@@ -95,7 +104,67 @@ public void generateSqlQueryForMasking() {
   @Override
   public void executeSqlQueryForMasking() {
     if (!dryRunEnabled) {
-      log.info("Executing script in order to create view in the database.");
+      log.trace("Invoking Redshift masking service to run data masking sql script.");
+      Properties connectionProps = new Properties();
+      connectionProps.setProperty("user", config.getSuperUser());
+      connectionProps.setProperty("password", config.getSuperUserPassword());
+      connectionProps.setProperty("db", config.getDatabase());
+      connectionProps.setProperty("schema", SNOWFLAKE_DEFAULT_SCHEMA);
```

Comment:
There is already a variable named `DEFAULT_INPUT_TABLE_SCHEMA` in DataMasking class

Reply:
replaced